### PR TITLE
Standardized Readme

### PR DESCRIPTION
**Pull request by RichardLitt**
See multiformats/multiformats#13

### CODE REVIEW

**Comment by jbenet** [comment]
LGTM, thanks!

The licenses on existing things should probably either stay with their authors for now (particularly those of non-PL people). new things can be PL.

File: README.md [hunk 1]
```diff
@@ -146,3 +164,20 @@ These are visual aids that help tell the story of why Multihash matters.
...
+
+Contributions welcome. Please check out [the issues](https://github.com/multiformats/multihash/issues).
+
+Check out our [contributing document](https://github.com/multiformats/multiformats/blob/master/contributing.md) for more information on how we work, and about contributing in general. Please be aware that all interactions related to multiformats are subject to the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+Small note: If editing the Readme, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
+
+## License
+
+[MIT](LICENSE) © Protocol Labs Inc.
```
Comment (jbenet) [inline README.md]: - sometimes the copyright notice is longer than one line, or it's complicated. or it's socially contentious.
- would be GREAT not to have to show this here and defer to the `LICENSE` file, is there a reason you want it here?

File: README.md [hunk 2]
```diff
@@ -146,3 +164,20 @@ These are visual aids that help tell the story of why Multihash matters.
...
+>>>>>>> Standardized Readme
```
Comment (jbenet) [inline README.md]: this looks like a merge artifact? maybe not, not sure. or is this a "standard-readme signature"

Comment (RichardLitt) [inline README.md]: Merge artifact. Good catch.

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: No reason. Will remove.

**Pull request merged**